src/EA.Weee.Web/Areas/Producer/ViewModels/ServiceOfNoticeViewModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace EA.Weee.Web.Areas.Producer.ViewModels
+{
+    using System;
+
+    public class ServiceOfNoticeViewModel
+    {
+        public Guid OrganisationId { get; set; }
+
+        public Guid DirectRegistrantId { get; set; }
+
+        public bool SameAsOrganisationAddress { get; set; }
+
+        public Core.Shared.AddressPostcodeRequiredData Address { get; set; }
```

Comment:
May  have to create an alternative version of this as it seems the service of notice address doesnt have an email :/ Which obviously has an impact elsewhere.